sycl/doc/PreprocessorMacros.md

```diff
@@ -48,6 +48,13 @@ This file describes macros that have effect on SYCL compiler and run-time.
   support for `assert()` via `aspect::ext_oneapi_native_assert`.
   This macro is undefined by default.
 
+- **__SYCL_DISABLE_PARALLEL_FOR_RANGE_ROUNDING__**
+
+  Defining this macro disables the automatic generation of a range rounded
+  kernel for `sycl::range` kernels. The macro must be defined before
+  `#include <sycl/sycl.hpp>`. Using this macro may reduce device binary size if
+  `sycl::range` kernels are used.
+
```

Comment:
User's are supposed to set command like flag to disable the optimization - `-fsycl-disable-range-rounding`.
We should not recommend defining this macro manually to avoid problems with wrong order of "define" and "include" directives.

I suggest we do not add this text.

Reply:
Have removed, thanks